src/train-classifier/src/test/resources/benchmarks_VerifyTrainClassifier.csv

```diff
@@ -0,0 +1,113 @@
+name, value, precision, higherIsBetter
+GradientBoostedTreesClassification_PimaIndian.csv_AUROC, 0.681730769230769, 0.01, true
+GradientBoostedTreesClassification_PimaIndian.csv_AUPR, 0.6758328422366278, 0.01, true
+MultilayerPerceptronClassifier_PimaIndian.csv_AUROC, 0.5, 0.01, true
+MultilayerPerceptronClassifier_PimaIndian.csv_AUPR, 0.6845425867507886, 0.01, true
+RandomForestClassification_PimaIndian.csv_AUROC, 0.8288461538461543, 0.01, true
+RandomForestClassification_PimaIndian.csv_AUPR, 0.7225351272502225, 0.01, true
+DecisionTreeClassification_PimaIndian.csv_AUROC, 0.6195299145299146, 0.01, true
+DecisionTreeClassification_PimaIndian.csv_AUPR, 0.5574243067309479, 0.01, true
+LogisticRegression_PimaIndian.csv_AUROC, 0.5, 0.01, true
+LogisticRegression_PimaIndian.csv_AUPR, 0.6845425867507886, 0.01, true
+NaiveBayesClassifier_PimaIndian.csv_AUROC, 0.38431623931623937, 0.01, true
+NaiveBayesClassifier_PimaIndian.csv_AUPR, 0.30590923737343967, 0.01, true
+GradientBoostedTreesClassification_data_banknote_authentication.csv_AUROC, 0.9790730757711891, 0.01, true
```

Comment:
I'd prefer to see learner, dataset and metric as separate strings instead of just one mega string

Reply:
unfortunately this really complicates things downstream, I thinks its better to have something simple and robust and leave more complex data comparisons as a good getting started project for a new member